Managers/CameraMan.cpp

```diff
@@ -258,7 +242,26 @@ namespace RTE {
 			newOffset += scrollVec * scrollProgress;
 		}
 
-		newOffset += screenShakeOffset;
+		if (g_ActivityMan.GetActivity()->GetActivityState() == Activity::ActivityState::Running) {
+			// Don't let our screen shake beyond our max.
+			screen.ScreenShakeMagnitude = std::min(screen.ScreenShakeMagnitude, m_ScreenShakeDecay * m_MaxScreenShakeTime);
+
+			// Reduce screen shake over time.
+			screen.ScreenShakeMagnitude -= m_ScreenShakeDecay * static_cast<float>(screen.ScrollTimer.GetElapsedSimTimeS());
```

Comment:
This should use RealTime, not SimTime, otherwise the screen shakes for an agonizingly long time when the game is lagging.